diff -up libcap-2.16/progs/capsh.c.chroot libcap-2.16/progs/capsh.c
--- libcap-2.16/progs/capsh.c.chroot	2011-08-11 13:34:32.000000000 +0200
+++ libcap-2.16/progs/capsh.c	2011-08-11 13:37:41.000000000 +0200
@@ -243,10 +243,16 @@ int main(int argc, char *argv[], char *e
 		perror("unable to lower CAP_SYS_CHROOT");
 		exit(1);
 	    }
+        /*
+         * Given we are now in a new directory tree, its good practice
+         * to start off in a sane location
+         */
+        status = chdir("/");
+
 	    cap_free(orig);
 
 	    if (status != 0) {
-		fprintf(stderr, "Unable to chroot to [%s]", argv[i]+9);
+		fprintf(stderr, "Unable to chroot/chdir to [%s]", argv[i]+9);
 		exit(1);
 	    }
 	} else if (!memcmp("--secbits=", argv[i], 10)) {
